docs/Functional Documentation/Functional_Specification.md

```diff
@@ -81,6 +81,50 @@ The codebase will be written in JavaScript as per decision made in [ADR-001](htt
 
 #### 1.4 Wallets
 
+**Modules called:**
+   - Crypto functions that generate pk, pubk, and pem 

```

Comment:
Shall we clarify terms? For example it might be not clear the difference between `pk` and `pubk`